examples/playground/json-examples/3d-heatmap.json

```diff
@@ -43,5 +43,13 @@
         [209, 55, 78]
       ]
     }
+  ],
+  "widgets": [
```

Comment:
Very nice, we should probably add these to more examples.

Reply:
Sure thing!